kubernetes/deployment-production.tmpl

```diff
@@ -127,6 +127,8 @@ data:
   STORAGE_ADAPTER: aws
   STORAGE_BUCKET: zooniverse-static
   STORAGE_PREFIX: panoptes-uploads.zooniverse.org/production/
+  AZURE_STORAGE_ACCOUNT: panoptes
+  AZURE_STORAGE_CONTAINER: production
```

Comment:
this needs to be update for public / private containers (e.g. staging deployment config)